docs/conf.py

```diff
@@ -118,14 +118,24 @@
             "name": "Donate to NumFocus",
         },
     ],
-    "github_url": "https://github.com/pydata/pydata-sphinx-theme",
-    "twitter_url": "https://twitter.com/PyData",
+    # "github_url": "https://github.com/pydata/pydata-sphinx-theme",
+    # "twitter_url": "https://twitter.com/PyData",
     "header_links_before_dropdown": 4,
     "icon_links": [
+        {
+            "name": "Twitter",
+            "url": "https://twitter.com/PyData",
+            "icon": "fa-brands fa-twitter",
+        },
+        {
+            "name": "GitHub",
+            "url": "https://github.com/pydata/pydata-sphinx-theme",
+            "icon": "fa-brands fa-github",
+        },
         {
             "name": "PyPI",
             "url": "https://pypi.org/project/pydata-sphinx-theme",
-            "icon": "fa-solid fa-box",
+            "icon": "fa-brands fa-python",
```

Comment:
IMO we should revert this. If you look at https://pypi.org/, its favicon is just a light grey cube. I think that's why the prior icon choice was a plain box. Ideally there would be a very small version of https://pypi.org/static/images/logo-small.2a411bc6.svg (or a simplified version of it) but I looked around for one about six months ago and couldn't find one. I think it's a little misleading to use the Python logo there though; that's the logo for python generally, not for PyPI.